lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -167,6 +167,10 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
   @override
   EngineFlutterView? view({required int id}) => viewManager[id];
 
+  @override
+  int get focusedViewId => _focusedViewId;
+  int _focusedViewId = -1;
```

Comment:
Is it too late for us to switch to an extension type that wraps the `int`? `int` is actually not a great type to express "ID" because they offer operations that make no sense for IDs. For example, what does the following expression mean?

```
var someViewId = PlatformDispatcher.instance.focusedViewId * 6;
```

We could hide the numeric nature of the ID behind an extension type, or behind a regular Dart class (e.g. `class ViewId { final int _id; }`.